WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupConfirmationFragment.kt

```diff
@@ -76,6 +76,7 @@ class SignupConfirmationFragment : Fragment() {
             mPhotoUrl = it.getString(ARG_SOCIAL_PHOTO_URL)
             mService = it.getString(ARG_SOCIAL_SERVICE)
         }
+        @Suppress("DEPRECATION")
         setHasOptionsMenu(true)
```

Comment:
Build fails on wpandroid with deprecations, so I suppressed this one as I'm not planning to rewrite the option menu